demos/vc_issuer/src/consent_message.rs

```diff
@@ -0,0 +1,107 @@
+//! This module contains the various consent messages that is displayed to the user when they are asked to consent to the issuance of a credential.
+
+use crate::SupportedCredentialType;
+use std::fmt::{Display, Formatter};
+use vc_util::issuer_api::{
+    Icrc21ConsentInfo, Icrc21ConsentMessageResponse, Icrc21ConsentPreferences,
+};
+use SupportedCredentialType::{UniversityDegreeCredential, VerifiedEmployee};
+use SupportedLanguage::{English, German};
+
+const EMPLOYMENT_VC_DESCRIPTION_ENG: &str = r###"# DFINITY Foundation Employment Credential
```

Comment:
nit:

```suggestion
const EMPLOYMENT_VC_DESCRIPTION_ENG: &str = r###"# DFINITY Foundation Employment Credential
```

to avoid confusion with ENGineer and for consistency with DE below

Reply:
Ah, yes. Much better 👍